### PR TITLE
docs: add copy button to spectron and devtron code blocks

### PR DESCRIPTION
**Pull request by stevejoachim**
See #4617 

image

### CODE REVIEW

**Review by erickzhao — COMMENTED** [review]
screencast 2020-10-09 11-13-57

Getting a weird visual bug on Chrome and FF on macOS (not Safari). 🤔

**Comment by stevejoachim** [comment]
Thanks for catching this! I was able to reproduce this on Firefox. It looks like the Copy button is trying to display a tooltip, but it gets trapped inside the code box and overflows it. I changed some CSS properties for the `pre` and `code` tags to fix this so that the tooltip will display (overflow: visible), but the actual code will still be scrollable (overflow: auto) if the window gets very narrow.

**Review by erickzhao — CHANGES_REQUESTED** [review]
image

Seems like it's gotten visually weirder with the latest push 🤔

**Comment by stevejoachim** [comment]
Thanks for reviewing. That's unfortunate that it's still messed up. What's your test setup? I can't reproduce that visual on Firefox, Safari, or Chrome on MacOS. This is what it's looking like in Firefox for me when hovering on one of the copy buttons:
image

File: public/styles/helpers/_code.scss [hunk 1]
```diff
@@ -54,16 +54,16 @@ h4 {
 
   pre {
     padding: $spacer-3;
-    overflow: auto;
+    overflow: visible;
```
Comment (vhashimotoo) [inline public/styles/helpers/_code.scss:57]:
```suggestion
	position: relative;
    overflow: visible;
```

File: public/styles/helpers/_code.scss [hunk 2]
```diff
@@ -54,16 +54,16 @@ h4 {
...
     font-size: 16px;
     line-height: 1.45;
 
     code {
-      display: inline;
+      display: block;
```
Comment (vhashimotoo) [inline public/styles/helpers/_code.scss:62]:
```suggestion
      display: inline;
```

File: public/styles/helpers/_code.scss [hunk 3]
```diff
@@ -54,16 +54,16 @@ h4 {
...
       max-width: initial;
       padding: 0;
       margin: 0;
-      overflow: initial;
+      overflow: scroll;
```
Comment (vhashimotoo) [inline public/styles/helpers/_code.scss:66]:
```suggestion
      overflow: initial;
```

**Review by vhashimotoo — CHANGES_REQUESTED** [review]
I'm also able to repro the same behavior as @erickzhao on Chrome, Windows 10. The suggestions change behavior to the same as we have at the `/docs` page.

**Comment by stevejoachim** [comment]
@vhashimotoo Thanks for reviewing the code in such detail. I just made the changes you suggested.

**Pull request closed without merging**